app/utils/additionalData/functions.ts

```diff
@@ -210,13 +225,60 @@ export const readJsonFileAndAddAdditionalData = (res: any) => {
   });
 };
 
+const getDeviceDetails = async () => {
+  return {
+    deviceBrand: getBrand(),
+    deviceModel: getModel(),
+    deviceManufacturer: await getManufacturer(),
+    deviceSystemName: getSystemName(),
+    deviceSystemVersion: getSystemVersion(),
```

Comment:
If we want to track this data from the user's device we also should add the usage to the Apple Privacy Information in AppStore Connect as well as in our own privacy policy.

Reply:
But bugsnag send all these details with more details, is that an issue too?

Reply:
Ok, depends on how to interpret this source https://developer.apple.com/app-store/app-privacy-details/ - as the above list does not contain any Device ID I think we might be fine.